tests/storage/storage.py

```diff
@@ -219,3 +219,34 @@ def xva_export_import(vm: VM, compression: XVACompression):
         if imported_vm is not None:
             imported_vm.destroy()
         vm.host.ssh(f'rm -f {xva_path}')
+
+ImageFormat = Literal['vhd', 'qcow2']
```

Comment:
This might have its place in `lib/vdi.py`?

Reply:
It would be nice indeed, but validating the type from xe requires some extra work that I began in https://github.com/xcp-ng/xcp-ng-tests/pull/317/commits, and that I might re-open